content/en/about/feature-stages/index.md

```diff
@@ -69,18 +69,18 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Stable
-| [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/)        | Stable
+| [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Deprecated
```

Comment:
nit: Can we sort the list to make it look better? Either by the feature name or by the phase should work.

Reply:
I reordered it based on phase.